### ISSUE TITLE
Apply templates doesn't apply the templates

### ISSUE THREAD

**Issue by Alexandermay** [curationexperts/mira#317]
This issue first noticed in the Tufts Dev. environment.

From batch review, after selecting the items I want to apply a template to, and then select  "Apply templates" nothing happens.  (I was expecting it to then offer  me the choice of overlay or replace, and then the option to select which option to apply.) From a "blank" search, after selecting the items I want to apply a template to, and then selecting the choice of overlay, nothing happens.  When I review the batch, it indicates the action failed.

**Comment by mark-dce** [comment]
Regression test once Batch Template apply is complete.

**Comment by acozine** [comment]
Tested with new MIRA code 4/30/15. Apply Template buttons provides expected behavior. There's a layout issue on the page where you select Preserve or Overwrite - I'll open a separate ticket for that.
